src/lib.rs

```diff
@@ -59,15 +59,11 @@ pub struct WeightedUtxo {
 #[cfg_attr(docsrs, doc(cfg(feature = "rand")))]
 pub fn select_coins<T: Utxo>(
     target: Amount,
-    cost_of_change: u64,
+    _cost_of_change: u64,
```

Comment:
What? Why is this not used?

Reply:
It's only used by bnb, not srd and frankly the current version of bnb that's in master is broken so I just commented it out.  I'm still waiting to get some review on https://github.com/p2pderivatives/rust-bitcoin-coin-selection/pull/30, especially those that created issues.

Reply:
Weird that it doesn't take into account the cost of change (why would anyone want such algo then?) anyway, just remove the argument?